DB/schema.sql

```diff
@@ -0,0 +1,43 @@
+
+
+CREATE TABLE MusicAlbum(
+  id INTEGER GENERATED ALWAYS AS INDENTITY PRIMARY KEY,
+  on_spotify BOOLEAN,
+  publish_date DATE
+);
+
+CREATE TABLE Genre(
+  id INTEGER GENERATED ALWAYS AS INDENTITY PRIMARY KEY,
+  name VARCHAR(150),
+  FOREIGN KEY(item_id) REFERENCES items(id)
+);
+
+CREATE TABLE Game (
+    id INTEGER GENERATED ALWAYS AS INDENTITY PRIMARY KEY,
+    last_played_date DATE NOT NULL,
+    multiplayer BOOLEAN NOT NULL,
+    FOREIGN KEY(item_id) REFERENCES items(id)
+);
+
+CREATE TABLE Author (
+    id INTEGER GENERATED ALWAYS AS INDENTITY PRIMARY KEY,
+    first_name VARCHAR(50),
+    last_name VARCHAR(50)
+);
+
+CREATE TABLE Book(
+  id INTEGER GENERATED ALWAYS AS INDENTITY PRIMARY KEY,
+  publish_date DATE,
+  archived BOOLEAN,
+  publisher VARCHAR(150),
+  cover_state VARCHAR(200),
+);
+
+CREATE TABLE Labels(
+  id INTEGER GENERATED ALWAYS AS INDENTITY PRIMARY KEY,
+  title VARCHAR(150),
+  color VARCHAR(100),
+  FOREIGN KEY(item_id) REFERENCES items(id)
+);
+

```

Comment:
- [ ] Great work so far on adding the tables and foreign keys 👍 but it would be nice if you add indexes to all the foreign keys to make your database efficient which you have done already many times 😊